hibernate-rx-core/src/test/java/org/hibernate/rx/containers/PostgreSQLDatabase.java

```diff
@@ -0,0 +1,39 @@
+package org.hibernate.rx.containers;
+
+import org.testcontainers.containers.PostgreSQLContainer;
+
+public class PostgreSQLDatabase {
+
+	private static final boolean USE_DOCKER = Boolean.getBoolean( "docker" );
+
+	/**
+	 * Holds configuration for the PostgreSQL database contianer. If the build is run with <code>-Pdocker</code> then
+	 * Testcontianers+Docker will be used.
+	 * <p>
+	 * TIP: To reuse the same containers across multiple runs, set `testcontainers.reuse.enable=true` in a file located
+	 * at `$HOME/.testcontainers.properties` (create the file if it does not exist).
+	 */
+	public static final PostgreSQLContainer<?> postgresql = new PostgreSQLContainer<>()
+			.withUsername( "hibernate-rx" )
+			.withPassword( "hibernate-rx" )
+			.withDatabaseName( "hibernate-rx" )
+			.withReuse( true );
+
+	public static String getJdbcUrl() {
+		if ( USE_DOCKER ) {
+			postgresql.start();
```

Comment:
I think you should add a comment to explain that `postgresql.start();` is required or  `postgresql.getJdbcUrl()` will throw an exception.